lab_1_classify_by_unigrams/main.py

```diff
@@ -85,12 +128,34 @@ def compare_profiles(
     'freq' in arguments, None is returned
     """
 
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict):
+        return None
+    if 'name' not in unknown_profile or 'name' not in profile_to_compare:
+        return None
+    if not all(isinstance(key, str) for key in unknown_profile) \
+            or not all(isinstance(key, str) for key in profile_to_compare) \
+            or not isinstance(unknown_profile['freq'], dict) \
+            or not isinstance(profile_to_compare['freq'], dict):
+        return None
+
+    all_chars = set(unknown_profile['freq']).union(set(profile_to_compare['freq']))
+
+    unknown_profile_freq = unknown_profile['freq']
```

Comment:
u can create this variable before the creating `all_chars` list and then use this variables in `all_chars` list creating

Reply:
done